maro/rl/policy/__init__.py

```diff
@@ -6,11 +6,13 @@
 from .dqn import DQN, PrioritizedExperienceReplay
 from .pg import PolicyGradient
 from .policy import AbsPolicy, NullPolicy, RLPolicy
```

Comment:
How about renaming `NullPolicy` to `DummyPolicy` ? @ysqyang  @buptchan

Reply:
I agree.

Reply:
I will change it in the base branch.